scripts/homestead.rb

```diff
@@ -141,7 +141,7 @@ def Homestead.configure(config, settings)
       end
 
       if (type == "symfony")
```

Comment:
you should change this to `symfony2`, to keep backwards compatibility with people using `symfony2` as type